### DISCUSSION TITLE
UN Datathon 2023 Pitch Plan

### DISCUSSION THREAD

**Discussion by ShweataNHegde** [petermr/petermr#75] [Planning]
## Our principles
- Guided by the UN principles for the UN Datathon: *Climate change, Biodiversity Loss* and Pollution
- Accessible to wide range of citizens including high schoolers and non-native English speakers
- Create tools to automate the creation of knowledge from heterogeneous sources and add semantics and ontology

## Examples for the hackathon
- Climate justice -- Is climate justice knowledge and research globally equitable? 
- Semantic IPCC-based multilingual glossary for climate using Wikidata 

## How We Work
- Constant flow of young learners in India
- Synchronous and async work (including a remote team)
- Goal-oriented rather than pedagogical
- Learning by doing
- Open to citizens

## We are looking for people
- with Climate Knowledge 
- with data visualization skills
- with experience building GUI
- who test our software
- who can help us create multilingual outputs
- who are storytellers

**Comment by mrchristian** [comment]
Here we can add ideas for questions about Acting Locally that we could use in the UN Datathon.

**Comment by mrchristian** [comment]
As an example here are some questions related to City Climate Plans. https://semanticclimate.org/city-open-climate-reader/ here we used a FAQ from the IPCC Reports. The questions were:

1. Q1. What measures can be taken by urban centers to mitigate and cut down on their emissions?
2. Q2. What role do renewable energy sources play in city climate plans?
3. Q3. How can cities promote sustainable transportation options in their climate action plans?
4. Q4. What strategies do cities employ to adapt to the impacts of climate change?
5. Q5. What are some successful examples of cities implementing effective climate action plans?

WGIII - had a lot of content related to urban areas - https://www.ipcc.ch/report/ar6/wg3/

**Reply by gracejl19** [reply]
1. How can authorities engage and collaborate with local communities and stakeholders in the development of their climate action plans?
2. How do urban green spaces and biodiversity conservation factor into city climate plans? 
3. What are the economic benefits and job opportunities associated with implementing climate action plans in cities?
4. What are the primary challenges and barriers cities face when trying to implement climate action plans in megacities, and how can they overcome these obstacles?
5. What role could play smart technology and data-driven solutions in enhancing the effectiveness of climate plans?
6. What are somes existing strategies that can ensure that climate plans are equitable and prioritize vulnerable communities in the process?

**Comment by mrchristian** [comment]
WGII Adaptation - also has city related content - https://www.ipcc.ch/report/ar6/wg2/

**Comment by Jemy29** [comment]
Q1. How can cities do a vulnerability assessment of the localities and include the same in their climate report?
Q2. How can the cities contribute to reducing the carbon footprint of individuals?
Q3. How can the cities localise the adaptations for climate risks in their action plans?

**Comment by mrchristian** [comment]
FYI on the UN Datathon Challenge:

https://unstats.un.org/wiki/display/UNDatathon2023/UN+Datathon+Theme

Your challenge as participants of the UN Datathon

Your challenge as a participant of the UN Datathon is to create an innovative data solution which tackles local sustainable development challenges, and which leverages one or several of the six transitions. Everything happens somewhere. Therefore, by using geospatial or geocoded data and combining it with other data sources your solution should support local action and monitoring in an innovative way. And as we are all interconnected in this world, your solution – even though focusing on a local issue – should ideally be adaptable to other regions, and hence be scalable at the global level so that as many people as possible can benefit from your solution.

**Comment by Jemy29** [comment]
Intro to vulnerability assessments: Climate change is a complex issue with social and environmental dynamics at play. The marginalized communities at all times are disproportionately affected by the impact of climate change. Climate vulnerability assessment takes into consideration the different built environments within a city or region that are more susceptible to climate related hazards such as floods, drought, heat waves, deteriorating air quality , impact of climate change, assesses the capacity of the entity to cope with and adapt to these impacts and identify areas of heightened vulnerabilities. This assessment enables policymakers to make informed decisions, resource allocation, build resilient strategies and reduce the risks associated with climate change. Rothman and Robinson (1997), have described the evolution of integrated vulnerability assessments through the following attributes, from linear to more complex chain of analyses, strictly quantitative to a more mixed analysis, from being science-driven to policy-driven assessments and from assessments that are dictated to the users to those that involve the users in the process.

**Comment by mrchristian** [comment]
Climate Vunerability - Use case write-up

https://docs.google.com/document/d/16a_FS95Y3SqinNjm5S7B2mW3ihhF2pkON7GXoO8Uf0U/edit#heading=h.hp3m7yan6573

### Use case: Climate Vulnerability Frameworks

**Support a user of UN Data exploring a question with data analysis and retrieval of Open Access scientific literature in the form of dashboards and Open Science informed replicable workflows.**

1. A researchers is working with UN Data to ask questions about climate vulnerability of specific countries for floods and heat waves.
2. The researcher carries our analysis on UN Data: Flood data and extreme heat (heat wave) data World UN Environment Programme: Environment Situation Room https://wesr.unep.org/ 
3. The frameworks for climate vulnerability being used need to be identified, known as Climate Vulnerability Indexes. It is observed that there are different frameworks and indexes and that these frameworks are customised for different countries and regions.
    a. Climate Vulnerability Index (CVI), also referred to as Climate Change Vulnerability Index (CCVI)
    b. One example mentioned by the UN is ND-GAIN Country Index https://gain.nd.edu/our-work/country-index/ 
    c. Indian CVI - https://www.ceew.in/sites/default/files/ceew-study-on-climate-change-vulnerability-index-and-district-level-risk-assessment.pdf 
4. The user is research topics of Flooding and Heat Wave vulnerability and from the scientific literature - For floods 429 papers were found; with 125 papers found for heat waves. Papers retrieved from EuropePMC automatically using TDM. https://europepmc.org/ 
    a. Papers can be geo located
    b. The papers can be retrieved and downloaded automatically
    c. Data analysis can be carried out on the papers to ask further questions. For example are the authors of the papers from the countries being researched.
5. The scientific literature searches and data analysis use Open Science research methodologies, they are: replicable, reproducible, and transparent. All code, configuration, and content is stored in a Git repository under open licences for reuse.
6. Search and analysis can be carried out in two ways:
   a. As one off searches
   b. As dashboards that can be set to periodically search and make notification of new literature.

**Reply by ShweataNHegde** [reply]
Thanks!

**Reply by mrchristian** [reply]
I found UN Data we can use on the 'use case' specifically on hazards - From here: https://data.cdp.net/ 

This one: 2022 Cities Climate Hazards - contains hazards - fooding and extreme heat - https://data.cdp.net/Climate-Hazards/2022-Cities-Climate-Hazards/rdq4-d52n
